### PR TITLE
chore: update sidetree-fabric

### PR DESCRIPTION
**Pull request by fqutishat**
- Add auth token

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/features/setupfabric.feature [hunk 1]
```diff
@@ -64,7 +68,7 @@ Feature:
...
+    When fabric-cli is executed with args "file upload --url https://peer0-org1.trustbloc.local/.well-known/did-trustbloc --files ./fixtures/discovery-config/sidetree-fabric/config/testnet.trustbloc.local.json;./fixtures/discovery-config/sidetree-fabric/config/org1.trustbloc.local.json;./fixtures/discovery-config/sidetree-fabric/config/org2.trustbloc.local.json;fixtures/discovery-config/sidetree-fabric/config/org3.trustbloc.local.json --idxurl https://peer0-org1.trustbloc.local/file/${fileIdxID} --pwd pwd1 --nextpwd pwd2  --signingkeyfile ./fixtures/keys/key.pem --authtoken ${token_fileidx_w} --contentauthtoken ${token_wellknown_w} --noprompt"
```
Comment (bstasyszyn) [inline test/bdd/features/setupfabric.feature:71]: Do we really need two different tokens for this, i.e. one for the fileidx and one for the content upload?

Comment (fqutishat) [reply test/bdd/features/setupfabric.feature]: done

File: test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml [hunk 2]
```diff
@@ -6,3 +6,8 @@
 
 basePath: /sidetree/0.0.1/blockchain
 maxTransactionsInResponse: 10
+maxBlocksInResponse: 5
+
+authorization:
+  writeTokens:
```
Comment (bstasyszyn) [inline test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml:12]: This needs to be readTokens - blockchain doesn't use writeTokens. If we don't want to restrict access to /blockchain then we don't need any authorization.

File: test/bdd/fixtures/fabric/config/ledger/mychannel-filehandler-wellknown-config.json [hunk 3]
```diff
@@ -4,5 +4,8 @@
   "ChaincodeName": "file",
   "Collection": "consortium",
   "IndexNamespace": "file:idx",
-  "IndexDocID": ""
+  "IndexDocID": "",
+  "Authorization":{
+  "WriteTokens": ["fileidx_w"]
```
Comment (troyronda) [inline test/bdd/fixtures/fabric/config/ledger/mychannel-filehandler-wellknown-config.json:9]: missing indent?

File: test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml (hunk 2 above)
Comment (fqutishat) [reply test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml:12]: ok

Comment (fqutishat) [reply test/bdd/fixtures/fabric/config/ledger/mychannel-blockchainhandler-config.yaml]: done

**Pull request merged**